docs/quickstart.rst

```diff
@@ -100,69 +52,54 @@ the :ref:`name parameter <name-parameter>` to group all those requests under an
 Additionally we've declared a `on_start` method. A method with this name will be called for each simulated 
 user when they start. For more info see :ref:`on-start-on-stop`.
 
+.. code-block:: python
+
+    wait_time = between(5, 9)
+
+Our class defines a ``wait_time`` function that will make the simulated users wait between 5 and 9 seconds after each task 
+is executed. For more info see :ref:`wait-time`.
 
 Start Locust
 ============
 
-To run Locust with the above Locust file, if it was named *locustfile.py* and located in the current working
-directory, we could run:
+Put the above code in a file named *locustfile.py* in your current directory and run:
 
 .. code-block:: console
 
     $ locust
 
 
-If the Locust file is located under a subdirectory and/or named different than *locustfile.py*, specify
-it using ``-f``:
+If your Locust file is located somewhere else, you can specify it using ``-f``
 
 .. code-block:: console
 
     $ locust -f locust_files/my_locust_file.py
 
 
-To run Locust distributed across multiple processes we would start a master process by specifying
-``--master``:
-
-.. code-block:: console
-
-    $ locust -f locust_files/my_locust_file.py --master
-
```

Comment:
I do think we should mention how to run Locust distributed in the Quickstart docs, since it's such an important USP.

Reply:
it is, just further down

Reply:
Ah, sorry, I didn't realize that!